msteams-platform/concepts/build-and-test/apps-localization.md

```diff
@@ -8,40 +8,42 @@ ms.date: 05/15/2018
+If you are publishing the app to the store, you must be aware that localizing your AppSource listing is not yet supported. To support localized listings in the app store, you can add additional languages to your listing. The default language information you provide in [Partner Center](/office/dev/store/submit-to-appsource-via-partner-center) for your listing appears in the [AppSource website](https://appsource.microsoft.com/marketplace/apps?product=office%3Bteams&page=1) listing for your app. Currently, the default language is English.
 
-### Example of configuring localization
+### Configure localization
 
-To configure an additional language for your app, in [Partner Center](/office/dev/store/submit-to-appsource-via-partner-center), select both English and the additional language of the app. French is used in this example:
+To configure an additional language for your app, in [Partner Center](/office/dev/store/submit-to-appsource-via-partner-center), select both English and the additional language of the app. French is used as an additional language in the following example:
 
 1. Add English language
     * Fill in the app name.
     * Fill in a short description of the app in English.
     * Fill in the long description of the app in English.
-    * In the long description, please also add the line “This app is available in “French”.
-    * Upload the images of your app UI (in English).
-2. Add French language
+    * In the long description, add the line **This app is also available in French**.
+    * Upload the images of your app UI in English.
+1. Add French language
     * Fill in the app name.
     * Fill in a short description of the app in French.
     * Fill in the long description of the app in French.
-    * Upload the images of your app UI (in French).
+    * Upload the images of your app UI in French.
 
-The images you upload with the English language will be the ones used in AppSource.
+The images that you upload with the English language are used in AppSource.
 
-## Localizing the strings in your app manifest
+## Localize strings in your app manifest
 
-You must use the Microsoft Teams app schema v1.5+ to properly localize your app. You can do this by setting the `$schema` attribute in your manifest.json file to 'https://developer.microsoft.com/en-us/json-schemas/teams/v1.8/MicrosoftTeams.Localization.schema.json' and updating the 'manifestVersion' property to '1.7'.
+You must use the Microsoft Teams app schema v1.5 and later to localize your app. You can do this by setting the `$schema` attribute in your manifest.json file to `https://developer.microsoft.com/en-us/json-schemas/teams/v1.8/MicrosoftTeams.Localization.schema.json` and updating the `manifestVersion` property to `1.7`. 
 
 ### Example manifest.json change
 
+Following is an example for manifest.json change to localize your app:
```

Comment:
This doesn't look right as a change for `manifest.json`. Please review this with Allen.

Reply:
Updated